### PR TITLE
draw an image on canvas map

### PR DESCRIPTION
**Pull request by mewiemers**

### CODE REVIEW

File: components/canvas/Canvas.module.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.containermap {
```
Comment (fre-ben) [inline components/canvas/Canvas.module.css]: Use camelcase to make classes better readable
```suggestion
.containerMap {
```

File: components/canvas/Canvas.module.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  background-image: url(/Albert.svg);
+  background-repeat: no-repeat;
+  align-content: center;
+}
+
+.show {
+  display: none;
+}
+
+.backgroundcanvas {
```
Comment (fre-ben) [inline components/canvas/Canvas.module.css]: Use camelcase to make classes better readable
```suggestion
.backgroundCanvas {
```

**Review by fre-ben — APPROVED** [review]
LGTM! 🧙

File: components/canvas/Canvas.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import styles from "./Canvas.module.css";
+import { useEffect, useRef } from "react";
+
+function Canvas() {
+  const canvasRef = useRef(null);
+
+  useEffect(() => {
+    const ctx = canvasRef.current.getContext("2d");
```
Comment (mhstrkmp) [inline components/canvas/Canvas.tsx]:
```suggestion
    const context = canvasRef.current.getContext("2d");
```

**Review by mhstrkmp — COMMENTED** [review]
lgtm 🚀

**Pull request merged**